applications/ehr_query_llm/lmm/agent_framework_op.py

```diff
@@ -0,0 +1,218 @@
+# SPDX-FileCopyrightText: Copyright (c) 2022-2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import json
+import logging
+import os
+from queue import Queue
+from threading import Thread
+from time import sleep
+
+from agents.ehr import EHRAgent
+from agents.ehr_builder import EHRBuilderAgent
+from agents.selector import SelectorAgent
+from holoscan.core import Operator, OperatorSpec
+from utils.chat_utils import ChatHistory
+from utils.response_handler import ResponseHandler
+
+from operators.ehr_query_llm.message_handling import MessageReceiver, MessageSender
+
+
+class AgentFrameworkOp(Operator):
```

Comment:
Please add a short explanatory docstring in each operator/agent declaration in these files to aid readability:

```suggestion
class AgentFrameworkOp(Operator):
    """This operator is responsible for ..."""
```